setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="streusle",
+    version="4.1",
```

Comment:
How should the version be specified for a development version? Should this be 4.2alpha?

Reply:
It should comply with [PEP 440](https://www.python.org/dev/peps/pep-0440/#public-version-identifiers), so that would be `4.2a` for an alpha release or `4.2.dev1`, `4.2.dev2`, etc. for development releases. Should I change it to one of these? And do you think the data and the code should be versioned with the same number?

Reply:
4.2.dev1 is OK. Should I update this every time I push to the repository?

I haven't been distinguishing data and code versions, as the official releases contain changes to both.

Reply:
I don't think it needs to be updated on every push; only for releases that have some significance. To install a from specific commit, you can do, for example,

     pip install git+https://github.com/danielhers/streusle@27f4af830d4670ba228ce25aaf8bedd9dcb788a6